src/transformers/utils/quantization_config.py

```diff
@@ -388,11 +391,14 @@ def __init__(
         self.pad_token_id = pad_token_id
         self.disable_exllama = disable_exllama
```

Comment:
if you enable exllamav2 you should disable exllama, and vice-versa no?

Reply:
Yes, I did something similar. Note that because exllama is activated by default, it is a little bit tricky.